### PR TITLE
Sync with k8s v1.13.0-alpha.0

### PR DESCRIPTION
**Pull request by karataliu**
cc @andyzhangx @feiskyer

### CODE REVIEW

File: tests/k8s-azure/manifest/linux.json [hunk 1]
```diff
@@ -4,11 +4,11 @@
   "properties": {
     "orchestratorProfile": {
       "orchestratorType": "Kubernetes",
-      "orchestratorRelease": "1.11",
+      "orchestratorVersion": "1.12.0-alpha.1",
```
Comment (andyzhangx) [inline tests/k8s-azure/manifest/linux.json]: why is `1.12.0-alpha.1` here?

File: glide.yaml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 package: k8s.io/cloud-provider-azure
 import:
 - package: k8s.io/kubernetes
-  version: v1.12.0-alpha.0
+  version: v1.13.0-alpha.0
 - package: github.com/Azure/azure-sdk-for-go
-  version: v14.6.0
+  version: v19.0.0
```
Comment (andyzhangx) [inline glide.yaml]: why change version of `azure-sdk-for-go`?

Comment (feiskyer) [reply glide.yaml]: k/k has upgraded to v19.0.0

File: tests/k8s-azure/manifest/linux.json [hunk 3]
```diff
@@ -4,11 +4,11 @@
...
       "kubernetesConfig": {
           "useCloudControllerManager": true,
-          "customCcmImage": "gcrio.azureedge.net/google_containers/cloud-controller-manager-amd64:v1.12.0-alpha.0",
-          "customHyperkubeImage": "gcrio.azureedge.net/google_containers/hyperkube-amd64:v1.12.0-alpha.0",
+          "customCcmImage": "gcrio.azureedge.net/google_containers/cloud-controller-manager-amd64:v1.13.0-alpha.0",
+          "customHyperkubeImage": "gcrio.azureedge.net/google_containers/hyperkube-amd64:v1.13.0-alpha.0",
```
Comment (feiskyer) [inline tests/k8s-azure/manifest/linux.json]: should we keep those version same with orchestratorVersion?

File: tests/k8s-azure/manifest/linux.json (hunk 1 above)
Comment (karataliu) [reply tests/k8s-azure/manifest/linux.json]: This is what acs-engine supports

File: tests/k8s-azure/manifest/linux.json (hunk 3 above)
Comment (karataliu) [reply tests/k8s-azure/manifest/linux.json]: This is image version, should be latest for testing.
orchestratorVersion is acs-engine's config.

Comment (feiskyer) [reply tests/k8s-azure/manifest/linux.json]: Thanks

**Pull request merged**